### DISCUSSION TITLE
One more amateur question: games played on browsers

### DISCUSSION THREAD

**Discussion by wc-yaa** [NateIsStalling/Basic4GLj#47] [General]
Hello, one more question.
It seems that nowadays indie developers like to develop browser games very much. The following jam(contest) is an example. There are 8 entries in the jam. 3 entries are Windows games and 5 entries are browser games. Will a version of Basic4GL for Web be developed in the future? 

https://itch.io/jam/jam-for-all-basic-dialects-5/entries

**Comment by NateIsStalling** [comment]
Hi @wc-yaa - I believe there was some community interest in a Basic4GL for Web version, but I am not aware of any active projects to develop a web version at this time. 
Basic4GLj is currently focused on supporting game and desktop app development for Windows, MacOS, and Linux, but I am open to considering support for browser games in the future.

Side note, the jam you linked looks pretty cool - I'll try to keep an eye out for any upcoming BASIC jam events and try to join in when I can 🙂

**Reply by wc-yaa** [reply]
> but I am open to considering support for browser games in the future. 

Hi! Thanks for considering this idea. 

> Side note, the jam you linked looks pretty cool - I'll try to keep an eye out for any upcoming BASIC jam events and try to join in when I can

You want to develop an entry for this type of jam? That's great. Making an entry in Basic4GL for Java may be an effective promotion for this BASIC dialect. The next jam should be in hosted in June or July in this year.
